src/main/java/burger/list/List.java

```diff
@@ -0,0 +1,223 @@
+package burger.list;
+
+import java.util.ArrayList;
+
+import burger.BurgerException;
+import burger.task.Deadline;
+import burger.task.Event;
+import burger.task.Task;
+import burger.task.Todo;
+
+import static burger.Burger.printLine;
+import static burger.BurgerException.*;
+
+public class List {
+
+    private final ArrayList<Task> taskList;
```

Comment:
any reason why this is final? this will be changed during the course of time, so it does not make much sense to declare it as final.